hikari/impl/bot.py

```diff
@@ -207,6 +207,9 @@ class GatewayBot(traits.GatewayBotAware):
         Defaults to `hikari.intents.Intents.ALL_UNPRIVILEGED`. This allows you
         to change which intents your application will use on the gateway. This
         can be used to control and change the types of events you will receive.
+    chunk_members: builtins.bool
```

Comment:
Does it ever make sense to forcibly request chunks? I thought the current implementation only requested chunks if it was possible, in which case "forcing" it would break things?

Reply:
I can see the use case in which you want to do something with the members at startup, so you forcibly request the chunks and then act on that instead of waiting for it to be fully chunked and then getting them off cache or iterating over them through REST

Reply:
Should I rename it `auto_chunk_members` or change this to be a tribool?

Reply:
Would like to get @FasterSpeeding opinion on this. Whatever they say is probably what is best. What I posted was just a random idea I came up with while reviewing

Reply:
Any name works, `auto_chunk_members` sounds good